### PR TITLE
fix(cms): broken links

### PR DESCRIPTION
**Pull request by scarf005**
type checked in 0.7.3

### CODE REVIEW

File: cms/configuration/storage.md [hunk 1]
```diff
@@ -76,8 +77,9 @@ GitHub API. It's useful if you want to allow people without GitHub accounts or
 permission to edit the site data.
 
 ```ts
-import lumeCMS, { GitHub } from "lume/cms.ts";
-import { Octokit } from "npm:octokit";
+import lumeCMS from "lume/cms/mod.ts";
+import GitHub from "lume/cms/storage/github.ts";
+import { Octokit } from "https://esm.sh/@octokit/rest";
```
Comment (oscarotero) [inline cms/configuration/storage.md:82]: why the switch to esm.sh?

Comment (scarf005) [reply cms/configuration/storage.md:82]: https://github.com/octokit/rest.js?tab=readme-ov-file#restjs

it was specified for `Browser` in `octokit/rest`, could revert to npm

Comment (oscarotero) [reply cms/configuration/storage.md:82]: I'd use NPM, basically because esm.sh use to break the lock file due different code versions. NPM is more stable.

Comment (scarf005) [reply cms/configuration/storage.md]: force-pushed in 0b694b8

**Comment by oscarotero** [comment]
Thank you!

**Pull request merged**